scripts/gatling/lib/solr.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+function solr_checkout_source_in() {
+  if [[ -z $1 ]]; then
+    >&2 echo "Required argument 'location' was not provided"
+    return 1
+  fi
+
+  local location=$1
+  if [[ -d $location/solr ]]; then
+    # The checkout already exists I guess?
+    return 0
+  fi
+
+  pushd $location
+    git clone git@github.com:apache/solr.git
+  popd
+}
+
+# Returns with an exit status indicating whether Solr is (or isn't)
+# running at the specified port.  Callers provide the port as a required
+# argument.  Assumes that the wording directory is positioned so that
+# 'bin/solr' can be invoked.
+#   Usage: solr_is_running 8983
+function solr_is_running() {
+  if [[ -z ${1:-} ]]; then
+    >&2 echo "Required argument 'port' was not provided"
+    return 1
+  fi
+
+  local port=$1
+  bin/solr assert --started "http://localhost:$port/solr"
+  return $?
+}
+
+function solr_kill_all() {
+  ps -ef | grep solr | grep java | awk {'print $2'} | xargs kill -9
+}
+
+# Start Solr (with custom ops pulled from START_SOLR_OPTS)
+# Assumes cwd of the Solr package root
+function solr_start() {
+  bin/solr start ${START_SOLR_OPTS:-}
```

Comment:
```suggestion
  bin/solr start -c ${START_SOLR_OPTS:-}
```

Reply:
we are going to want to add some backcompat in solr 10 so -c doesn't raise an error I bet!